README.md

```diff
@@ -3,7 +3,7 @@
 [![GitHub Actions CI Status](https://github.com/nf-core/cutandrun/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/cutandrun/actions?query=workflow%3A%22nf-core+CI%22)
 [![GitHub Actions Linting Status](https://github.com/nf-core/cutandrun/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/cutandrun/actions?query=workflow%3A%22nf-core+linting%22)
 [![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/cutandrun/results)
-[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
+[![DOI](http://img.shields.io/badge/DOI-10.5281/zenodo.5653536-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.5653536)
```

Comment:
```suggestion
[![DOI](http://img.shields.io/badge/DOI-10.5281/zenodo.5653535-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.5653535)
```

Reply:
The version we want to use is the one that says "Cite all versions" so we don't have to update the Zenodo DOI after every release.

Reply:
How come this changed to a 5 instead of 6 at the end?

Reply:
Because of the all versions thing..